app/views/layouts/_navigation.html.erb

```diff
@@ -0,0 +1,31 @@
+<nav class="bg-black">
+  <div class="container mx-auto px-4 py-5 flex items-center justify-between">
+    <ul class="flex space-x-4">
+      <li><a href="/v1/client/products" class="text-white hover:text-gray-200 font-bold text-lg">Dashboard</a></li>
+      <li><a href="/v1/client/products" class="text-white hover:text-gray-200 font-bold text-lg"><small>Products</small></a></li>
+      <li><a href="/v1/client/orders" class="text-white hover:text-gray-200 font-bold text-lg"><small>Orders History</small></a></li>
```

Comment:
Don't you use link_to?

Reply:
yeah i use "link_to" as well, over here to get things done speedily i just placed in an href, if you want "link_to" then i can do that as well.